packages/cli/package.json

```diff
@@ -19,7 +25,6 @@
     "typecheck": "tsc",
     "checks": "pnpm typecheck",
     "build": "rm -rf lib && esbuild src/cli.ts --outdir=lib --bundle --format=esm --packages=external",
-    "local-run": "tsx --no-warnings ./src/bin.ts",
```

Comment:
what's the replacement for this? `NODE_OPTIONS='--conditions=webstudio --import=tsx' webstudio` is this something we need to memorize to run it localy?

Reply:
or did you want to set NODE_OPTIONS into the .env by default?

Reply:
> something we need to memorize to run it localy?

not memoize, copy paste
local-run script is useless it is in cli package